tests/running/math.lua

```diff
@@ -215,6 +215,8 @@ function test17()
            math.max(3, 3.0, 3.0) == 3 and
            is_integer(math.max(3, 3.0, 3.0)) and
            math.max(-5, -4, -3, -2, -1, 0, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1) == 10
+    -- Tests we do not currently pass due to incompatibility with PUC-Rio Lua
+    --     is_err(math.max("1"))
```

Comment:
In PUC-Rio Lua 5.3.5, it seems like `math.max("1")` returns `1`, does luster return error here?

Reply:
Sorry, I didn't actually check the behavior of that, I just assumed it errored.  In actuality, it's `math.max(1, "2", 1)` that errors in PUC-Rio Lua.  I'll adjust it.